src/robotide/ui/mainframe.py

```diff
@@ -37,15 +37,16 @@
 from .fileexplorerplugin import FileExplorer
 from .notebook import NoteBook
 from .progress import LoadProgressObserver
+from ..editor import customsourceeditor
 
 
 _menudata = """
 [File]
 !&New Project | Create a new top level suite | Ctrlcmd-N
 ---
 !&Open Test Suite | Open file containing tests | Ctrlcmd-O | ART_FILE_OPEN
-!Open &Directory | Open directory containing datafiles | Shift-Ctrlcmd-O | \
-ART_FOLDER_OPEN
+!Open &Directory | Open directory containing datafiles | Shift-Ctrlcmd-O | ART_FOLDER_OPEN
+!Open External File | Open file in Code Editor | | ART_FOLDER_OPEN
```

Comment:
For now, "Open &Directory" & "Open External File" are using the exact same icon in menu bar, can we update to a new icon?